gradle/compatibility.gradle

```diff
@@ -8,7 +8,7 @@ apiValidation {
         'ktor-client-json-tests',
         'ktor-client-content-negotiation-tests',
         'ktor-client',
-        'ktor-client-features',
+        'ktor-client-plugins',
```

Comment:
consider keeping old modules(empty with transitive dependency to the renamed module)

Reply:
Will be added in the next PR after all moving will be ready